src/screens/UsersScreen.tsx

```
@@ -0,0 +1,134 @@
+import {
+  View,
+  Text,
+  StyleSheet,
+  FlatList,
+  TouchableOpacity,
+  RefreshControl,
+} from 'react-native';
+import {User} from '../types';
+
+const UsersScreen: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
```

ESLint

[error] 'setUsers' is assigned a value but never used. Allowed unused elements of array destructuring patterns must match /^_/u.

(@typescript-eslint/no-unused-vars)